frontend/views/pages/Home.vue

```diff
@@ -80,6 +80,9 @@ export default {
     SvgCreateGroup
   },
   mounted () {
+    console.log('::groupId:', this.$store.state.currentGroupId)
+    console.log('::groupsByName:', this.$store.getters.groupsByName)
```

Comment:
Should be deleted unless needed. If needed, reasoning needed, and remember `console.debug`.

Reply:
removed